types/vegapulse/vegapulseAir42/distance.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$id": "https://akenza.io/vega/vegapulsAir42/distance.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "distance",
+  "title": "Distance",
+  "properties": {
+    "distance": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/distance/schema.json#/$defs/distance/cm"
+    },
+    "fillLevel": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/distance/schema.json#/$defs/fillLevel/percent"
+    },
+    "unit": {
+      "title": "Unit",
+      "description": "Unit indicator",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 255,
+      "hideFromKpis": true
+    },
+    "inclinationDegree": {
+      "title": "Inclination degree",
+      "unit": "°",
+      "type": "number",
+      "description": "Inclination degree"
+    },
+    "detailedState": {
+      "title": "Detail state",
+      "type": "integer",
```

Comment:
should this be a string?

Reply:
Lets leave it as a number as i do not find a translation for it-